packages/libffi/project.bri

```diff
@@ -29,6 +29,21 @@ export default function (): std.Recipe<std.Directory> {
   });
 }
 
+export async function test() {
+  const script = std.runBash`
+    echo -n $(pkg-config --modversion libffi) | tee "$BRIOCHE_OUTPUT"
```

Comment:
This is a great alternative to look for the library version. Instead of relying on building a small software that tries to print the version.

Sometimes, libraries such as libffi do not expose capabilities to retrieve the current version installed.

Reply:
Ooh cool!

At some point I want to add a `test` library or something, with a bunch of utilities for writing common tests / assertions / utilities